test/fuzz/test_runner.py

```diff
@@ -150,7 +150,7 @@ def run_once(*, corpus, test_list, build_dir, export_coverage, use_valgrind):
             corpus_path,
```

Comment:
Unrelated to this pull, but it why is detect_leaks disabled?

I'd suggest to include this commit:

```
commit ffff1245c11428968b71454e4ef85417d33c9893 (HEAD)
Author: MarcoFalke <falke.marco@gmail.com>
Date:   Fri Jan 3 10:27:04 2020 -0800

    test: Enable leak detection for fuzzers
    
    This has been disabled in 16f0a186dcee563bb1000e1ffc51da87e7623bc6 for
    no given reason.

diff --git a/test/fuzz/test_runner.py b/test/fuzz/test_runner.py
index b638e6bac6..5174e21e2a 100755
--- a/test/fuzz/test_runner.py
+++ b/test/fuzz/test_runner.py
@@ -146,7 +146,6 @@ def run_once(*, corpus, test_list, build_dir, export_coverage, use_valgrind):
         args = [
             os.path.join(build_dir, 'src', 'test', 'fuzz', t),
             '-runs=1',
-            '-detect_leaks=0',
             corpus_path,
         ]
         if use_valgrind:

Reply:
`-detect_leaks=0` was required at some points to make `test/fuzz/test_runner.py` pass. If that is not required any more that is great news! I'll try if Travis is happy without it.